staging/src/k8s.io/apimachinery/pkg/apis/config/v1alpha1/defaults.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	kruntime "k8s.io/apimachinery/pkg/runtime"
+)
+
+func addDefaultingFuncs(scheme *kruntime.Scheme) error {
```

Comment:
This is something we will discuss if we're actually gonna do, but for now this is ok